src/ExpressionEvaluator/Package/ExpressionEvaluatorPackage.csproj

```diff
@@ -41,48 +48,76 @@
       <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup</IncludeOutputGroupsInVSIX>
       <IncludeOutputGroupsInVSIXLocalOnly>DebugSymbolsProjectOutputGroup</IncludeOutputGroupsInVSIXLocalOnly>
       <ForceIncludeInVSIX>true</ForceIncludeInVSIX>
+      <Ngen>true</Ngen>
```

Comment:
Microsoft.CodeAnalysis.ExpressionEvaluator.ExpressionCompiler.dll

---
In reply to: [91194161](https://github.com/dotnet/roslyn/pull/15710#discussion_r91194161) [](ancestors = 91194161)